src/modules/Elsa.MassTransit.AzureServiceBus/Notifications/CleanupSubscriptions.cs

```diff
@@ -0,0 +1,8 @@
+using Elsa.Mediator.Contracts;
+
+namespace Elsa.MassTransit.AzureServiceBus.Notifications;
+
+/// <summary>
+/// Notification to clean up the Azure Service Bus subscriptions.
+/// </summary>
+public record CleanupSubscriptions : INotification;
```

Comment:
This sounds more like a command than an event. Unless there are multiple handlers interested in this, it might make more sense to turn this into a command.